framework/src/modules/chain/modules/accounts.js

```diff
@@ -65,7 +65,15 @@ class Accounts {
 			transactionTypes.VOTE
 		] = library.logic.transaction.attachAssetType(
 			transactionTypes.VOTE,
-			new Vote(scope.logger, library.schema, library.logic.account)
+			new Vote({
+				components: {
+					logger: scope.logger,
+				},
+				libraries: {
+					schema: library.schema,
+					account: library.logic.account,
```

Comment:
Related to the previous comment.

Reply:
I agree with you but this is happening on the account module not in logic so it is out of the scope of this PR.